plugins/__init__.py

```diff
@@ -1093,3 +1094,76 @@ def is_base64_encoded(s: "string") -> "bool":
     except Exception:
         return False
     return True
+
+
+@plugin
+def hostname(fqdn: "string") -> "string":
+    """
+    Return the hostname part of the fqdn
+    """
+    return fqdn.split(".")[0]
+
+
+@plugin
+def netmask(prefixlen: "int") -> "std::ipv4_address":
+    """
+    Given the cidr, return the netmask
+    """
+    interface = ipaddress.ip_interface(f"255.255.255.255/{prefixlen}")
+    return str(interface.netmask)
+
+
+@plugin
+def cidr_to_prefixlen(addr: "std::ipv_any_interface") -> "int":
```

Comment:
```suggestion
def prefixlen(addr: "std::ipv_any_interface") -> "int":
```

I would not add the 'cidr_to', I don't think these is a lot of chance at confusion